AirCasting/APICommunicator/StreamDownloader.swift

```diff
@@ -0,0 +1,74 @@
+// Created by Lunar on 29/03/2022.
+//
+
+import Foundation
+import Resolver
+
+struct StreamWithMeasurementsDownstream: Decodable {
```

Comment:
a little naming issue: `Downstream` is usually used as a suffix for the `protocol` that is responsible for getting data, not data itself. This one can be named either `StreamWithMeasurements` or `StreamWithMeasurementsDownstreamData`